lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -507,15 +507,21 @@ states:
     nestedJourney: ADDRESS_AND_FRAUD
     exitEvents:
       next:
-        targetState: CRI_NINO_J6
+        targetState: CRI_DWP_KBV_J7
         checkIfDisabled:
-          hmrcKbv:
-            targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
+          dwpKbv:
+            targetState: CRI_NINO_J6
+            checkIfDisabled:
+              hmrcKbv:
+                targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
       enhanced-verification:
         targetState: CRI_NINO_J6
         checkIfDisabled:
-          hmrcKbv:
-            targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
+          dwpKbv:
+            targetState: CRI_DWP_KBV_J7
```

Comment:
Should this be `CRI_NINO_J6`?
I think this is saying "if dwpkbv is disabled then go down the DWP KBV journey"